code/src/shared/i18n/I18nProvider.tsx

```diff
@@ -8,7 +8,7 @@ type I18nProividerProps = {
 
 const I18nProivider: FC<I18nProividerProps> = ({ children }) => {
   return (
-    <IntlProvider locale="en" defaultLocale="en" messages={messages.en}>
+    <IntlProvider locale="uk" defaultLocale="uk" messages={messages.uk}>
```

Comment:
Why default locale is changes from `en` to `uk`?

Reply:
Very nice observation, thanks. I changed it to test the website with ukrainian language, but forgot to change it back. Fixed.